src/main/index.ts

```diff
@@ -37,30 +38,54 @@ let windowManager: WindowManager;
 
 app.setName(appName);
 
+logger.info("📟 Setting as Lens as protocol client for lens://");
```

Comment:
```suggestion
logger.info("📟 Setting Lens as protocol client for lens://");
```

Reply:
Good catch, though I will do that in a follow up PR so as to not need new approves.